### PR TITLE
Corrections to trust blog

### PR DESCRIPTION
**Pull request by natebower**
### Description
[Corrections to content that was added subsequent to editorial review.]

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2024-01-24-trust-in-open-source-software.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ While transparency is a key pillar of open-source projects, questions regarding
...
+We were most interested in measuring trust in OpenSearch. The sample indicated a 7.4 out of 10 average score regarding trust. This measure comprised the 6 measures. We used the K D Paine & Partners 6-factor scale. Aspects of this measure included perceived dependability (4-item measure; 7.8; alpha =.89), perceived sense of control mutuality (3-item measure; 7.4; alpha = .96), perceived commitment (5-item measure; 7.5; alpha = .94), satisfaction, (5-item measure; 7.1, alpha = .95) communal relationship quality (2-item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5). Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development).
```
Comment (natebower) [inline _posts/2024-01-24-trust-in-open-source-software.md:32]: It's not clear what we mean by "This measure comprised the 6 measures", and the phrase is redundant. Which measures? If we mean the ones referenced in the following sentence, we can just delete this sentence and say "We used the K D Paine & Partners 6-factor scale for this survey." However, I can't find any real references to this organization/scale online. Is the naming of the organization and the scale correct as written?

**Comment by natebower** [comment]
@krisfreedain @apasun Please make these corrections to the content that was added subsequent to editorial review. Thanks!

File: _posts/2024-01-24-trust-in-open-source-software.md (hunk 1 above)
Comment (apasun) [reply _posts/2024-01-24-trust-in-open-source-software.md:32]: This measure comprised of 6 dimensions

Comment (natebower) [inline _posts/2024-01-24-trust-in-open-source-software.md:32]:
```suggestion
We were most interested in measuring trust in OpenSearch. The sample indicated a 7.4 out of 10 average score regarding trust. This measure is comprised of 6 dimensions. We used the K D Paine & Partners 6-factor scale. Aspects of this measure included perceived dependability (4-item measure; 7.8; alpha =.89), perceived sense of control mutuality (3-item measure; 7.4; alpha = .96), perceived commitment (5-item measure; 7.5; alpha = .94), satisfaction, (5-item measure; 7.1, alpha = .95) communal relationship quality (2-item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5). Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development). 
```

Comment (apasun) [reply _posts/2024-01-24-trust-in-open-source-software.md:32]: https://www.instituteforpr.org/wp-content/uploads/2003_MeasuringTrust.pdf

Comment (natebower) [reply _posts/2024-01-24-trust-in-open-source-software.md:32]: @apasun I have made the necessary changes based on your feedback. Thanks!

**Comment by natebower** [comment]
Now just need an approval @krisfreedain. Thanks!

File: _posts/2024-01-24-trust-in-open-source-software.md [hunk 2]
```diff
@@ -22,14 +22,14 @@ While transparency is a key pillar of open-source projects, questions regarding
...
+We were most interested in measuring trust in OpenSearch. The sample indicated a 7.4 out of 10 average score regarding trust. This measure is comprised of six dimensions. We used the KDPaine & Partners six-component scale. Aspects of this measure included perceived dependability (4-item measure; 7.8; alpha =.89), perceived sense of control mutuality (3-item measure; 7.4; alpha = .96), perceived commitment (5-item measure; 7.5; alpha = .94), satisfaction, (5-item measure; 7.1, alpha = .95) communal relationship quality (2-item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5). Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development).
```
Comment (natebower) [inline _posts/2024-01-24-trust-in-open-source-software.md:32]:
```suggestion
We were most interested in measuring trust in OpenSearch. The sample indicated a 7.4 out of 10 average score regarding trust. This measure is comprised of six dimensions. We adopted the KDPaine & Partners scale for measuring trust. Aspects of this measure included perceived dependability (4-item measure; 7.8; alpha =.89), perceived sense of control mutuality (3-item measure; 7.4; alpha = .96), perceived commitment (5-item measure; 7.5; alpha = .94), satisfaction, (5-item measure; 7.1, alpha = .95) communal relationship quality (2-item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5). Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development). 
```

File: _posts/2024-01-24-trust-in-open-source-software.md [hunk 3]
```diff
@@ -22,14 +22,14 @@ While transparency is a key pillar of open-source projects, questions regarding
...
+We were most interested in measuring trust in OpenSearch. The sample indicated a 7.4 out of 10 average score regarding trust. This measure is comprised of six dimensions. We adopted the KDPaine & Partners scale for measuring trust. Aspects of this measure included perceived dependability (4-item measure; 7.8; alpha =.89), perceived sense of control mutuality (3-item measure; 7.4; alpha = .96), perceived commitment (5-item measure; 7.5; alpha = .94), satisfaction, (5-item measure; 7.1, alpha = .95) communal relationship quality (2-item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5). Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development).
```
Comment (natebower) [inline _posts/2024-01-24-trust-in-open-source-software.md:32]:
```suggestion
We were most interested in measuring trust in OpenSearch. The sample indicated a 7.4 out of 10 average score regarding trust. We adopted the KDPaine & Partners scale for measuring trust. This scale is comprised of six dimensions. Aspects of this measure included perceived dependability (4-item measure; 7.8; alpha =.89), perceived sense of control mutuality (3-item measure; 7.4; alpha = .96), perceived commitment (5-item measure; 7.5; alpha = .94), satisfaction, (5-item measure; 7.1, alpha = .95) communal relationship quality (2-item measure; 7.5, alpha = .77) and, importantly, exchange relationship quality (single item; 7.5). Depending on community perception, an open-source project can [enhance trust in multiple ways](https://www.linuxfoundation.org/blog/12-ways-to-improve-the-effectiveness-and-impact-of-enterprise-open-source-development). 
```

**Review by krisfreedain — APPROVED** [review]
thanks @natebower

**Pull request merged**